src/main/java/net/dries007/tfc/common/blocks/devices/PowderkegBlock.java

```diff
@@ -121,4 +122,25 @@ public boolean canDropFromExplosion(BlockState state, BlockGetter level, BlockPo
     {
         return false;
     }
+
+    @Override
+    public int getAnalogOutputSignal(BlockState state, Level level, BlockPos pos)
+    {
+        return level.getBlockEntity(pos, TFCBlockEntities.POWDERKEG.get()).map(InventoryBlockEntity::getAnalogValue).orElse(super.getAnalogOutputSignal(state, level, pos));
+    }
+
+    @Override
+    public boolean hasAnalogOutputSignal(BlockState state)
+    {
+        return true;
+    }
+
+    @Override
+    public void neighborChanged(BlockState state, Level level, BlockPos pos, Block block, BlockPos neighborPos, boolean s)
+    {
+        if (!level.isClientSide && level.hasNeighborSignal(pos))
+        {
+            toggleSeal(level, pos, state);
```

Comment:
Moreover, if we do add config options, one per device ("barrel reacts to redstone", "powderkeg reacts to redstone") I think would be good. For some reason I think the comparator output is a fairly no-brainer but the redstone input I could see being controversial.